CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [unreleased]
 
 -   removed jsonschema dependency
+-   emailDelivery and smsDelivery Ingredients added
+-   SMTP service added for emailDelivery
+-   Twilio service added for smsDelivery
```

Comment:
Deprecated use of older functions. Name those functions and describe what needs to be implemented instead. Be as clear as you can be in this. As an example, see how https://github.com/supertokens/supertokens-auth-react/blob/web-js-integration/CHANGELOG.md is done